src/data/faq_de.json

```diff
@@ -117,6 +117,62 @@
                     }
                 ]
             },
+            {
+                "title": "Testzertifikat",
+                "id": "test_cert",
+                "accordion": [
+                    {
+                        "title": "Wozu dient das digitales COVID-Testzertifikat?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Ab der Version 2.4 der Corona-Warn-App (CWA) können Sie ein digitales Testzertifikat über die App anfordern.  Durch die neue Funktion haben Sie die Möglichkeit, ihr negatives Testergebnis durch ein offizielles, digitales COVID-Testzertifikat in Form eines QR-Codes in der CWA nachzuweisen.  Innerhalb der Länder der Europäischen Union kann es als offiziell gültiger Nachweis eines negativen Testergebnisses dienen, den Sie beispielsweise für Reisen nutzen können oder in anderen gesetzlich vorgeschriebenen Fällen, wie z.B. einem Restaurantbesuch. Sie können dann den QR-Code in der CWA vorzeigen."
+                        ]
+                    },
+                    {
+                        "title": "Wie kann ich ein Testzertifikat anfordern?",
+                        "anchor": "test_cert_get",
+                        "active": true,
+                        "textblock": [
+                            "Das Zertifikat können Sie anfordern, indem sie einen Test in der Corona-Warn-App (CWA) registrieren: Dazu können sie auf dem Starbildschirm ihrer CWA auf 'Test registrieren' tippen und dann auf 'QR-Code scannen. Nachdem sie den QR-Code zur Test-Registrierung gescannt haben, öffnet sich automatisch das Fenster zum COVID-Testzertifikat. Dort können sie das 'Testzertifikat anfordern' oder ablehnen, falls sie keins erhalten möchten."

```

Comment:
After scannen is a ' missing.
All  sie have to be Sie.

Reply:
Also "ihrer" => "Ihrer"

Reply:
ok